README.md

```diff
@@ -113,10 +115,33 @@ additional metrics!
 
 See the [`docs`](docs) directory for more information on the exposed metrics.
 
+#### Conflict resolution in label names
+
+`*_labels` family of metrics exposes Kubernetes labels as Prometheus labels.
+[Kubernetes](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
+is more liberal than
+[Prometheus](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels)
+in terms of allowed characters in label names,
+so we automatically convert unsupported characters to underscores,
+e.g. `app.kubernetes.io/name` becomes `label_app_kubernetes_io_name`.
```

Comment:
```suggestion
For e.g. - `app.kubernetes.io/name` becomes `label_app_kubernetes_io_name`.
```

Reply:
This is the only change I'm not sure about as it would basically read "For for example".
Let me change it to simply say "For example".